ossec_security/assets/logs/ossec-security_tests.yaml

```
@@ -0,0 +1,429 @@
+    result:
+      custom:
+        action: "IMPORT"
+        component: "ub20->/root/test.log"
+        description: "User uploaded a file to server."
+        group: " syslog,ftpd,"
+        group_list:
+        - "syslog"
+        - "ftpd"
+        level: 1
+        local_file_path: "/opt/test.txt"
+        log: "May 28 19:38:24 ub20 ftpd[24474]: FTPD: IMPORT file local /opt/test.txt, remote"
+        rule_id: 11104
+        service: "ftpd"
+        status: "informational"
+        syslog:
+          appname: "ossec"
+          hostname: "ub20"
+        timestamp: 45645542000
+      message: "<132>Jun 13 12:49:02 ub20 ossec: {\"crit\":1,\"id\":11104,\"component\":\"ub20->/root/test.log\",\"classification\":\" syslog,ftpd,\",\"description\":\"User uploaded a file to server.\",\"message\":\"May 28 19:38:24 ub20 ftpd[24474]: FTPD: IMPORT file local /opt/test.txt, remote\"}"
+      service: "ftpd"
+      status: "info"
+      tags:
+      - "source:LOGS_SOURCE"
+      timestamp: 45645542000
+  -
+    sample: "<132>Jun 13 11:38:38 ub20 ossec: {\"crit\":10,\"id\":11109,\"component\":\"ub20->/root/test.log\",\"classification\":\" syslog,ftpd,authentication_failures,\",\"description\":\"Multiple FTP failed login attempts.\",\"message\":\"Jun  1 16:16:26 ub20 ftpd[39056]: repeated login failures from 10.10.10.10\",\"src_ip\":\"10.10.10.10\"}"
+    result:
+      custom:
+        component: "ub20->/root/test.log"
+        description: "Multiple FTP failed login attempts."
+        group: " syslog,ftpd,authentication_failures,"
+        group_list:
+        - "syslog"
+        - "ftpd"
+        - "authentication_failures"
+        level: 10
+        log: "Jun  1 16:16:26 ub20 ftpd[39056]: repeated login failures from 10.10.10.10"
+        network:
+          client:
+            geoip: {}
+            ip: "10.10.10.10"
+        rule_id: 11109
+        service: "ftpd"
+        status: "error"
+        syslog:
+          appname: "ossec"
+          hostname: "ub20"
+        timestamp: 45641318000
+      message: "<132>Jun 13 11:38:38 ub20 ossec: {\"crit\":10,\"id\":11109,\"component\":\"ub20->/root/test.log\",\"classification\":\" syslog,ftpd,authentication_failures,\",\"description\":\"Multiple FTP failed login attempts.\",\"message\":\"Jun  1 16:16:26 ub20 ftpd[39056]: repeated login failures from 10.10.10.10\",\"src_ip\":\"10.10.10.10\"}"
+      service: "ftpd"
+      status: "error"
+      tags:
+      - "source:LOGS_SOURCE"
+      timestamp: 45641318000
+  -
+    sample: "<132>Jun 13 11:48:52 ub20 ossec: {\"crit\":5,\"id\":11107,\"component\":\"ub20->/root/test.log\",\"classification\":\" syslog,ftpd,access_denied,\",\"description\":\"Connection blocked by Tcp Wrappers.\",\"message\":\"Jun  3 02:32:37 ub20 ftpd[418042]: refused connect from 10.10.10.10\",\"src_ip\":\"10.10.10.10\"}"
+    result:
+      custom:
+        component: "ub20->/root/test.log"
+        description: "Connection blocked by Tcp Wrappers."
```

```suggestion
        description: "Connection blocked by TCP Wrappers."
```
Some descriptions have periods at the end and some don't, if you'd like to make them consistent

---

those description field is something which is retrieved from events received from OSSEC. Hence, modifying them is not recommended.